### DISCUSSION TITLE
How to name a minecraft server?

### DISCUSSION THREAD

**Discussion by Capri205** [GameServerManagers/LinuxGSM#3492] [General]
POS. Thanks.

**Comment by wldwalker** [comment]
Edit the server.properties file and you will find you can change your name in that file.

**Reply by Capri205** [reply]
No. Not what I mean. Say I run a lobby server and some game servers on the same host:

/home/mcserver/lobby/...
/home/mcserver/skyblock/...
/home/mcserver/bedwars/...

How do you differentiate between servers so that the command "./mcserver start" in each directory doesn't start a server called "NOT SET". Starting the lobby means you can't start skyblock or bedwars since you get the error "server NOT_SET" is already running.

**Reply by boatplugs** [reply]
I believe you can copy the ./mcserver binary as a different filename and it should start a separate instance.
